tests/unit/checkDeployBlockersTest.ts

```diff
@@ -79,25 +88,27 @@ function mockIssue(prList, deployBlockerList) {
 **Compare Changes:** https://github.com/Expensify/App/compare/production...staging
 
 **This release contains changes from the following pull requests:**
-${_.map(
-    prList,
-    ({url, isQASuccess}) => `
+${prList
+    .map(
+        ({url, isQASuccess}) => `
 - ${checkbox(isQASuccess)} ${url}
 `,
-)}
+    )
+    .join('\n')}
 ${
-    !_.isEmpty(deployBlockerList)
+    !deployBlockerList || deployBlockerList.length < 0
         ? `
 
 **Deploy Blockers:**`
         : ''
 }  
```

Comment:
NAB: Since you didn't introduce this code but there's some stray whitespace here 

```suggestion
}
```